### PR TITLE
docs(metrics): reformat serviceMonitor Helm variable descriptions

### PR DESCRIPTION
**Pull request by ejweber**
#### Which issue(s) this PR fixes:

longhorn/longhorn#8142

#### What this PR does / why we need it:

Final description formatting per @jillian-maroket's review of https://github.com/longhorn/website/pull/945.

### CODE REVIEW

**Comment by ejweber** [comment]
@mergify backport v1.7.x v1.6.x

**Review by jillian-maroket — DISMISSED** [review]

File: chart/README.md [hunk 1]
```diff
@@ -232,12 +232,12 @@ You can install Longhorn in an air-gapped environment with a private registry. F
...
 | metrics.serviceMonitor.interval | string | `""` | Interval at which Prometheus scrapes the metrics from the target. |
-| metrics.serviceMonitor.metricRelabelings | list | `[]` | Configures the relabeling rules to apply to the samples before ingestion. See Prometheus operator documentation for format. |
-| metrics.serviceMonitor.relabelings | list | `[]` | Configures the relabeling rules to apply the target’s metadata labels. See Prometheus operator documentation for format. |
+| metrics.serviceMonitor.metricRelabelings | list | `[]` | Configures the relabeling rules to apply to the samples before ingestion. See the [Prometheus Operator  documentation](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.Endpoint) for formatting details. |
+| metrics.serviceMonitor.relabelings | list | `[]` | Configures the relabeling rules to apply the target’s metadata labels. See the [Prometheus Operator  documentation](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.Endpoint) for formatting details. |
```
Comment (james-munson) [inline chart/README.md:240]: Nit: extra space in "Operator  documentation"

**Review by james-munson — DISMISSED** [review]
One nit, looks good.

File: chart/README.md (hunk 1 above)
Comment (ejweber) [reply chart/README.md]: Thank you. Fixed.

**Review by james-munson — APPROVED** [review]

**Pull request merged**